libs/utils/src/Card/web_service_card.ts

```diff
@@ -104,4 +109,37 @@ export class WebServiceCard implements Card {
       throw new Error('Failed to write long value');
     }
   }
+
+  /**
+   * Writes new short and long values to the card.
+   */
+  async writeShortAndLongValues({
+    shortValue,
+    longValue,
+  }: ShortAndLongValues): Promise<void> {
+    const { success } = (await fetchJson('/card/write_short_and_long', {
+      method: 'post',
+      headers: {
+        'Content-Type': 'application/x-www-form-urlencoded;charset=UTF-8',
+      },
+      body:
+        `short_value=${encodeURIComponent(shortValue)}&` +
+        `long_value=${encodeURIComponent(longValue)}`,
+    })) as SuccessIndicationResponse;
+    if (!success) {
+      throw new Error('Failed to write short and long values');
+    }
+  }
+
+  /**
+   * Overrides card write protection.
+   */
+  async overrideWriteProtection(): Promise<void> {
```

Comment:
I went back and forth on whether to implicitly override write protection within write methods or to have consumers explicitly do so. I landed on the latter as a safer, non-breaking change

Reply:
Seems easy enough to change later if we desire